### ISSUE TITLE
Converts long integers to e-notation

### ISSUE THREAD

**Issue by lucifer-crybaby** [UgnisSoftware/react-spreadsheet-import#136]
First of all great package

When I upload sequence of numbers either formatted as number or text in my office
![Screenshot from 2022-09-25 22-47-25](https://user-images.githubusercontent.com/66128532/192149948-1bf1afac-5ffe-4a24-952b-710b055d469b.png)

it converts it to e-notation
![Screenshot from 2022-09-25 22-47-12](https://user-images.githubusercontent.com/66128532/192149956-f0b01be7-21a7-441d-8d21-e1bbb5957270.png)

is there any way to bypass or deal with it ?

**Comment by gbrocha** [comment]
Any update here?

**Comment by JulitorK** [comment]
The parser (sheetjs) we use does this under the hood and follows ECMA-376 (the primary specification for XLSX) standard.   From document: 

> The application shall attempt to display the full number up to 11 digits (inc. decimal point). If the number is too large, the application shall attempt to show exponential format.
> 
> For general formatting in cells, max overall length for cell display is 11, not including negative sign, but includes leading zeros and decimal separator.  

![Screen_Shot_2023-09-04_at_9 51 51_AM](https://github.com/UgnisSoftware/react-spreadsheet-import/assets/45755753/c7dad612-41f0-40bb-b6ba-8c54df8e58d8)

One way to fix this is to edit excel file cell format from ‘General’ to ‘Number’.
Another is to save cell data as text by adding a quotation mark in front of it, e.g. `’99999`

We could also implement ability to save all numbers without formatting. Any currency, percentage or deliberate scientific notation formatting would be lost (e.g. `21%` -> `0.21`, `$888.00` -> `888`). If it covers your usecase it could be a solution.

Let me know if you want us to implement `rawNumbers` option.

**Comment by masiulis** [comment]
We have discussed this issue in detail, and we don't see a good solution to it. While @JulitorK has fixed the issues with `rawNumbers` in SheetJS, this option had a lot of side effects as mentioned above. For now we recommend formatting cells in Excel from `General` to `Number`.

We've also discussed exposing all SheetJS props, but that would connect our API to SheetJS API directly, which we would like to avoid. In case you want to change SheetJS parsing, we encourage forking the repository.
